### DISCUSSION TITLE
Modular Voice Chat mod

### DISCUSSION THREAD

**Discussion by miguelcarcamov** [itzg/docker-minecraft-server#1022] [Q&A]
Hello,

I am running an instance of Minecraft 1.12.2 and the latest version of Forge on AWS using the docker minecraft server. I want to use the Modular Voice Chat Mod version 1.1.1. I have installed the mod on the server and I have configurated the port in the mod config file, the docker compose and in the AWS config. However, all the players are still receiving the message: "Cannot connect to voice server, try reconnecting! (or see logs for complete errors). The log only shows this:
`
| [12:22:33] [Server thread/INFO] [STDOUT]: [fr.nathanael2611.modularvoicechat.util.Helpers:log:248]: [12:22] [ModularVoiceChat] Requesting carcamovski to connect to voice-server... Sending packet.`

Has anyone tried this mod the docker minecraft server?

Thanks!

**Comment by ikigai-gh** [comment]
@miguelcarcamov In order to connect to server you should open both TCP and UDP ports, see:
https://github.com/Nathanael2611/ModularVoiceChat/blob/784b5b9a4e243d318f1d1588a9a024ad5a713765/src/main/java/fr/nathanael2611/modularvoicechat/client/voice/VoiceClient.java#L54 
Here author uses kryonet library, which accepts ports as third and fourth parameters, therefore client crashes with an error ConnectionRefused
